src/components/AppHeader.vue

```diff
@@ -236,32 +260,22 @@ export default {
         this.setSelectedDevice(value);
       },
     },
-    startDateTime: {
+    rangeDateTime: {
       get() {
-        return moment
+        let startDateTime = moment
           .utc(this.$store.state.startDateTime, DATE_TIME_FORMAT)
           .local()
-          .format(DATE_TIME_FORMAT);
-      },
-      set(value) {
-        this.setStartDateTime(
-          moment(value, DATE_TIME_FORMAT).utc().format(DATE_TIME_FORMAT)
-        );
-      },
-    },
-    endDateTime: {
-      get() {
-        return moment
+          .toDate();
+        let endDateTime = moment
           .utc(this.$store.state.endDateTime, DATE_TIME_FORMAT)
           .local()
-          .format(DATE_TIME_FORMAT);
+          .toDate();
+        return [startDateTime, endDateTime];
       },
       set(value) {
+        this.setStartDateTime(moment(value[0]).utc().format(DATE_TIME_FORMAT));
```

Comment:
this would probably be more readable with `set([startDateTime, endDateTime]) {}` instead of `value[n]`